hypha/apply/todo/templates/todo/todolist_item.html

```diff
@@ -12,5 +12,8 @@
             </span>
         {% endif %}
     </div>
+    {% if task.user %}
+        <button class="ms-2 invisible group-hover:visible" data-tooltip="Remove" hx-delete="{% url 'todo:delete' pk=task.id %}" hx-target="this" hx-swap="none" hx-confirm='{% trans "Are you sure you want to remove this task? It will remove this task from your task list." %}'>{% heroicon_micro "trash" class="inline me-2" aria_hidden=true %}</button>
```

Comment:
```suggestion
        <button class="ms-2 sm:invisible sm:group-hover:visible" data-tooltip="Remove" hx-delete="{% url 'todo:delete' pk=task.id %}" hx-target="this" hx-swap="none" hx-confirm='{% trans "Are you sure you want to remove this task? It will remove this task from your task list." %}'>{% heroicon_micro "trash" class="inline me-2" aria_hidden=true %}</button>
```

Mobile users can't access the remove button if using a touch screen. Also worth making sure it plays well with #4057 and scales for all mobile screens. On the test instance it looked to break around a width of 384px